pkg/project/api/v1beta1/conversion.go

```diff
@@ -0,0 +1,80 @@
+package v1beta1
+
+import (
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/conversion"
+
+	newer "github.com/openshift/origin/pkg/project/api"
+)
+
+func init() {
+	err := kapi.Scheme.AddConversionFuncs(
+		func(in *newer.Project, out *Project, s conversion.Scope) error {
+			if in.Annotations != nil {
```

Comment:
move this block at the end of the function, most notably after the ObjectMeta conversion